web/src/components/ProjectInfo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+const ProjectWrapper = styled.div`
+  ${tw`container flex flex-col text-brand-9 lg:flex-row mb-24`}
```

Comment:
Add `lg:justify-between` for horizontal alignment/spacing

Reply:
@asasmith this would not take. i tried every which way, which is why I added the mr on the heading

Reply:
@asasmith I changed it but I still dont think this is actually doing anything

Reply:
@kcbreuil I think your issue was that you had `w-auto` on the body element so it was taking all of the available space. Updating to `w-2/3` seems to be working. 

with `w-auto`
![Screen Shot 2021-03-01 at 11 12 17 AM](https://user-images.githubusercontent.com/11489171/109524996-1499ef80-7a7f-11eb-856d-c8c6b4a8b3f4.png)

with `w-2/3`
![Screen Shot 2021-03-01 at 11 12 40 AM](https://user-images.githubusercontent.com/11489171/109525013-19f73a00-7a7f-11eb-8e8c-fde970c79947.png)